doc/decoding/decoding_intro.rst

```diff
@@ -463,13 +463,17 @@ We can try Fisher's `Linear Discriminant Analysis (LDA)
 
 Import the module::
 
-    >>> from sklearn.lda import LDA
+    >>> from sklearn.discriminant_analysis import LinearDiscriminantAnalysis  # doctest: +SKIP
```

Comment:
Could you add a comment (a rst comment, that is not seen in the rendered document) to remove the skips when we stop supporting sklearn versions older than 0.17